go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
```

Comment:
The package versions are specified with `// indirect`, indicating they are not directly imported in your application. It is advisable to periodically audit your dependencies to remove unused ones, improving both security and performance.